integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTwoDomainsLoadBalancers.java

```diff
@@ -1941,4 +1946,46 @@ private void checkIngressReady(boolean isHostRouting, String ingressHost, boolea
       assertTrue(callWebAppAndWaitTillReady(curlCmd, 60));
     }
   }
+
+  /**
+   * Create PV and PVC for Apache custom configuration file in specified namespace.
+   * @param apacheNamespace namespace in which to create PVC
+   */
```

Comment:
Don't we have common utility to create a generic PV/PVC. Check with Sankar

Reply:
I used the common utility to create PV/PVC:
createPVPVCAndVerify(v1pv, v1pvc, labelSelector, apacheNamespace);
We need to pass the V1PersistentVolume and V1PersistentVolumeClaim objects to that method.